exercises/practice/linked-list/.meta/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "blurb": "Implement a doubly linked list.",
+  "authors": [
+    "Atomic-Germ"
+  ],
+  "contributors": [],
```

Comment:
Suggestion: run `bin/configlet fmt -u -e $slug`

Reply:
I definitely thought I ran that 🧐